README.rst

```diff
@@ -17,23 +17,47 @@
 .. |black| image:: https://img.shields.io/badge/code%20style-black-000000.svg
 
 
-`skrub <https://skrub-data.github.io/>`_ (formerly *dirty_cat*) is a Python
+**skrub** (formerly *dirty_cat*) is a Python
 library that facilitates prepping your tables for machine learning.
 
 If you like the package, spread the word and ⭐ this repository!
 You can also join the `discord server <https://discord.gg/ABaPnm7fDC>`_.
 
-What can `skrub` do?
---------------------
-
-`skrub` provides data assembling tools (``TableVectorizer``, ``fuzzy_join``...) and
-encoders (``GapEncoder``, ``MinHashEncoder``...) for **morphological similarities**,
-for which we usually identify three common cases: **similarities, typos and variations**
+Website: https://skrub-data.org/
+
+What can skrub do?
+------------------
+
+The goal of skrub is to facilitate building and deploying machine-learning
+models on tabular data.
+
+skrub provides high-level tools for joining dataframes (``Joiner``, ``AggJoiner``, ...),
+encoding columns (``MinHashEncoder``, ``ToCategorical``, ...), building a pipeline
+(``TableVectorizer``, ``tabular_learner``, ...), and more.
+
+>>> from skrub.datasets import fetch_employee_salaries
+>>> dataset = fetch_employee_salaries()
+>>> df = dataset.X
+>>> y = dataset.y
+>>> df.iloc[0]
+gender                                                                     F
+department                                                               POL
+department_name                                         Department of Police
+division                   MSB Information Mgmt and Tech Division Records...
+assignment_category                                         Fulltime-Regular
+employee_position_title                          Office Services Coordinator
+date_first_hired                                                  09/22/1986
+year_first_hired                                                        1986
+
+>>> from sklearn.model_selection import cross_val_score
+>>> from skrub import tabular_learner
+>>> cross_val_score(tabular_learner('regressor'), df, y)
+array([0.89370447, 0.89279068, 0.92282557, 0.92319094, 0.92162666])
 
 See our `examples <https://skrub-data.org/stable/auto_examples>`_.
 
-What `skrub` cannot do
-~~~~~~~~~~~~~~~~~~~~~~
+What skrub cannot do
+~~~~~~~~~~~~~~~~~~~~
```

Comment:
that section seems more specific to stuff that used to be the focus of dirty_cat, in the context of skrub it seems a bit weird -- I wonder if we should drop the what skrub cannot do section

Reply:
Another argument in favor of removing is that it's rare to see a readme explicitly mention what a package *cannot* do

@glemaitre wdyt ?